microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -153,55 +195,77 @@ The CIDR notation for grouping IP addresses was removed from the `clusterNetwork
+* Previously, KCM and KAS did not exit after getting a cancellation signal. With this update, {microshift-short} shuts down as expected. (link:https://issues.redhat.com/browse/OCPBUGS-18773[*OCPBUGS-18773*])
 
-* Previously, CRI-O was not configured with the correct default path to pause the container because pods with `shareProcessNamespace` set to `true` could not start. With this update, CRI-O is configured explicitly with the correct path. (link:https://issues.redhat.com/browse/OCPBUGS-7874[*OCPBUGS-7874*])
+* Previously, kubeconfigs used a certificate authority (CA) bundle that included 3 different signers: service network, localhost, and external. This made kubeconfigs interchangeable and capable of validating for other networks. With this update, the CAs used to generate kubeconfig files for {microshift-short}'s embedded components were reconfigured to ensure the kubeconfigs are independent. (link:https://issues.redhat.com/browse/OCPBUGS-8301[*OCPBUGS-8301*])
 
-* Previously, the mDNS server in {microshift-short} was advertising all IP addresses on the host. As a result, clients relying on mDNS to resolve the hostname for a route might have been given an internal IP address that could not be used by an application running on {microshift-short}. With this update, the mDNS server in {microshift-short} advertises internal IP addresses. (link:https://issues.redhat.com/browse/OCPBUGS-7205[*OCPBUGS-7205*])
+[discrete]
+[id="microshift-4-14-networking-bug-fixes"]
+==== Networking
 
-* Previously, using a `.local` suffix for a hostname broke the name resolution with DNS. As a result, {microshift-short} was unable to reach its own hostname. With this update, mDNS was fixed to use IP addresses instead of hostnames whenever possible. (link:https://issues.redhat.com/browse/OCPBUGS-10766[*OCPBUGS-10766*])
+* Previously, mixing uppercase and lowercase letters in the hostname caused {microshift-short} to fail. This was because the node name was taken directly from the hostname, and Kubernetes does not use uppercase names. With this update, the hostname only uses lowercase host names. (link:https://issues.redhat.com/browse/OCPBUGS-8411[*OCPBUGS-8411*])
 
-* Previously, when IP addresses were used instead of hostnames to connect to the apiserver, external connectivity failed. As a result, certificates did not regenerate when the IP address was changed. The kubeconfig used names instead of IP addresses which required the use of `subjectAltNames` if IP addresses were alternativly used. With this update, the previous kubeconfigs for absent IP addresses and names are removed. (link:https://issues.redhat.com/browse/OCPBUGS-11395[*OCPBUGS-11395*])
+* Previously, the networking smoke test created a `hello-microshift` pod using `busybox` and an image pulled from Docker. With this update, quay.io is used to host the image. (link:https://issues.redhat.com/browse/OCPBUGS-19939[*OCPBUGS-19939*])
```

Comment:
This is a fix related to internal e2e tests. I'm not sure we'd want it in the list.